yarn.lock

```diff
@@ -9314,7 +9314,12 @@ typescript-json-schema@^0.53.0:
     typescript "~4.5.0"
     yargs "^17.1.1"
 
-typescript@^4.5.5, typescript@~4.5.0:
+typescript@^4.6.2:
+  version "4.6.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.6.2.tgz#fe12d2727b708f4eef40f51598b3398baa9611d4"
+  integrity sha512-HM/hFigTBHZhLXshn9sN37H085+hQGeJHJ/X7LpBWLID/fbc2acUMfU+lGD98X81sKP+pFa9f0DZmCwB9GnbAg==
+
+typescript@~4.5.0:
```

Comment:
```
yarn why typescript
...
=> Found "typescript-json-schema#typescript@4.5.5"
info This module exists because "_project_#block-template#typescript-json-schema" depends on it.
...
```

Should be fine till we can upgrade `typescript-json-schema`. Related PR: https://github.com/YousefED/typescript-json-schema/pull/469